### PR TITLE
Document how to run example in readme

### PR DESCRIPTION
**Pull request by rukai**
This took me a while to figure out, I'm sure others would find it useful in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,11 @@ Supported platforms:
 This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. You can test them with `make`.
 
 The idiomatic Rust wrapper lives in https://github.com/gfx-rs/wgpu-rs and provides a number of more complex examples to get a feel of the API.
+
+To run the C triangle example, run these commands at the root of the repo:
+```
+rustup toolchain install nightly-x86_64-unknown-linux-gnu
```
Comment (grovesNL) [inline README.md]: We shouldn't require a specific toolchain – this should work fine on macOS, for example

Comment (rukai) [reply README.md]: Oops forgot to expand on that.

Comment (rukai) [reply README.md]: I guess they need a c compiler as well....

File: README.md [hunk 2]
```diff
@@ -20,6 +20,13 @@ Supported platforms:
...
 ## Usage
 
-This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. You can test them with `make`.
+This repository contains C-language examples that link to the native library targets and perform basic rendering and computation.
+To run the C triangle example, install a C compiler, then run these commands at the root of the repo:
+```
+rustup toolchain install nightly
+make examples-native
+cd examples/hello_triangle_c/build/
+./hello_triangle
```
Comment (kvark) [inline README.md]: does it have to run from the `build` folder? maybe we can avoid `cd` altogether?

File: README.md [hunk 3]
```diff
@@ -20,6 +20,13 @@ Supported platforms:
 
 ## Usage
 
-This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. You can test them with `make`.
+This repository contains C-language examples that link to the native library targets and perform basic rendering and computation.
+To run the C triangle example, install a C compiler, then run these commands at the root of the repo:
+```
+rustup toolchain install nightly
+make examples-native
```
Comment (kvark) [inline README.md]: I think this would fail to run `cbindgen` if not installed

Comment (grovesNL) [reply README.md]: Maybe we should separate header generation out of this target

File: README.md (hunk 2 above)
Comment (grovesNL) [reply README.md]: Alternatively during the build we could include the shaders or set an absolute path for them

File: README.md (hunk 3 above)
Comment (rukai) [reply README.md]: It doesn't seem to complain at all without cbindgen installed.

**Comment by rukai** [comment]
I tried compiling on another machine and found that I need ccache installed, which is just bizarre.
Its quite possible the machine was just incorrectly setup and I don't need ccache, so please let me know if you find that to be the case.

File: README.md (hunk 3 above)
Comment (rukai) [reply README.md]: Turns out it is needed, so I added it.
I see how I got confused though.

File: README.md [hunk 4]
```diff
@@ -20,6 +20,14 @@ Supported platforms:
 
 ## Usage
 
-This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. You can test them with `make`.
+This repository contains C-language examples that link to the native library targets and perform basic rendering and computation.
+To run the C triangle example, install a C compiler + glfw 3, then run these commands at the root of the repo:
+```
+rustup toolchain install nightly
+cargo install cbindgen
+make examples-native
```
Comment (kvark) [inline README.md]: Sorry about the changes underneath, but there is no longer "examples-native".
Please update to latest folder structure, and we'll proceed.

Comment (rukai) [reply README.md]: `make examples-native` still works, I don't understand what your requesting?

**Comment by kvark** [comment]
Oh sorry, I got confused for a moment.
Bors r+

**Pull request merged**